### PR TITLE
Darksky weather api enricher

### PR DESCRIPTION
**Pull request by kootenpv**
The goal is to enable adding historic data. Not a lot of APIs offer (free) historical data for any location. But Darksky offers 1000 calls per day. And in this call, you get hourly data (so 24 of them). This is good enough granularity I believe.

In my case, to backfill all my data, I need 4 days.

Would still be nice to be able to easily schedule.

Also, need to think of a way to not make enrichers add to data, but enable some kind of virtual data that can be joined on.

### CODE REVIEW

**Review by jjc385 — DISMISSED** [review]
Looks pretty good.

Also, I really like the description/ discussion.  Though I think the commit message leaves a little to be desired.  Maybe "Add basics of Darksky weather api enricher" would be better?  Then probably "targets #4" in the message body.

**Comment by kootenpv** [comment]
@jjc385 True, it's a bad description, not sure where my head was.

When we squash we should make sure it mentions it also adds `iterate_time` and its functionality (namely, returning a single row for an evenly space timed).

**Comment by NickolayVasilishin** [comment]
> Looks pretty good.
> 
> Also, I really like the description/ discussion. Though I think the commit message leaves a little to be desired. Maybe "Add basics of Darksky weather api enricher" would be better? Then probably "targets #4" in the message body.

But you will squash all your commits here in github ui, so then you put here a meaningful commit message summary
So it doesn't really matter what you write in git messages, but it can just help you not rewriting everything on squash

File: nostalgia/enrichers/darksky_weather/api.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+from nostalgia.cache import get_cache
+
+CACHE = get_cache("darksky_weather")
+
+dotenv.load_dotenv(".env")
+dotenv.load_dotenv("~/nostalgia_data/.env")
+
+
+def _historic_weather(lat, lon, epoch_time):
+    q = f"{lat},{lon},{epoch_time}"
```
Comment (NickolayVasilishin) [inline nostalgia/enrichers/darksky_weather/api.py]: I would ask you to use as much verbose variable names as possible

**Review by NickolayVasilishin — DISMISSED** [review]

**Comment by kootenpv** [comment]
I also added a function that simply returns the data per hour of a day.

**Review by NickolayVasilishin — DISMISSED** [review]

**Comment by kootenpv** [comment]
Too bad it did not detect the review was succesful on the earlier commit

**Pull request merged**